go/enclave/rpc_server.go

```diff
@@ -266,3 +271,27 @@ func (s *server) decodeBlock(encodedBlock []byte) types.Block {
 	}
 	return block
 }
+
+// serializeEVMError deserializes EVM errors into the RPC response
+// always returns a SerialisableError byte slice
+func serializeEVMError(err error) ([]byte, error) {
+	var errSerializedBytes []byte
+	var errReturn interface{}
+	var marshallErr error
+
+	// it's a generic error, can't extract more info
+	errReturn = evm.SerialisableError{Err: err.Error()}
+
+	// it's already a serialized error, handle any error wrapping that might have occurred
```

Comment:
How does this block handle any wrapping?

Reply:
```
	var e evm.SerialisableError
	if ok := errors.As(err, &e); ok {
		errReturn = e
	}

	```
it converts the errReturn  to the Serialisable error even if err has wrapped it.